app/src/main/java/com/belatrixsf/allstars/ui/contacts/ContactFragment.java

```diff
@@ -129,7 +133,35 @@ protected void initDependencies(AllStarsApplication allStarsApplication) {
     public void onViewCreated(View view, Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
         initViews();
-        contactPresenter.getEmployeeList();
+        if (savedInstanceState != null) {
+            restoreState(savedInstanceState);
+            if (contactPresenter.isInActionMode()) {
+                contactFragmentListener.setActionMode(actionModeCallback);
+            }
```

Comment:
should call for a function on presenter and presenter should call the view function